resources/publications.md

```diff
@@ -95,12 +93,35 @@ td:nth-child(2) {
 
 ## Metrics
 
+### Learnable metrics
+
+Directly optimize correlation with human judgments.
+
 |     |     |     |
 | --- | --- | --- |
 | 2020 | [***COMET: A Neural Framework for MT Evaluation***](https://aclanthology.org/2020.emnlp-main.213.pdf) | Ricardo Rei, Craig Stewart, Ana C. Farinha, Alon Lavie |
+| 2020 | [***BLEURT: Learning Robust Metrics for Text Generation***](https://aclanthology.org/2020.acl-main.704.pdf) | Thibault Sellam, Dipanjan Das, and Ankur Parikh |
+
+### Embedding-based metrics
+
+Use various word embeddings as an alternative to n-gram matching for capturing word semantics similarity.
+
+|     |     |     |
+| --- | --- | --- |
+| 2020 | [***Bertscore: Evaluating text generation with bert***](https://openreview.net/forum?id=SkeHuCVFDr) | Tianyi Zhang, Varsha Kishore, Felix Wu, Kilian Q. Weinberger, and Yoav Artzi |
+| 2019 | [***MoverScore: Text generation evaluating with contextualized embeddings and earth mover distance***](https://aclanthology.org/D19-1053.pdf) | Wei Zhao, Maxime Peyrard, Fei Liu, Yang Gao, Christian M. Meyer, Steffen Eger |
+
+
+### n-gram matching metrics
```

Comment:
To me this seems like prescribing the approach.

Isn't the common denominator here mainly that they depend on a reference translation?

Reply:
No, I don't think so. From the contents of the section, I thought that here we are talking about Quality evaluation: this means that all the metrics use reference. This is roughly what I had in mind
<img width="744" alt="Screenshot 2022-04-07 at 14 27 17" src="https://user-images.githubusercontent.com/16096019/162209986-39d1089d-199f-4725-9ca0-1f41d2123b9e.png">

So what we could do is
1) instead of "Metrics" have separate sections for Quality estimation and Quality evaluation, link to the article on the website explaining the difference (or explain in one sentence and refer to the article for more details).
2) current content goes to Quality Evaluation
3) something (COMET-QE etc) goes to Quality Estimation.

Reply:
Btw, if no one is working on the Quality pages, I could take over these, research the topic more and add something like the figure above to the corresponding page.

Reply:
Ah, got it, thanks for explaining

Your suggestion for re-organising the sections makes sense to me.

Basically the dichotomy in the picture is good, but it doesn't really come through on the page right now.

So let's go with your suggestion.

And we should be really careful to keep the wording simple.

Reply:
> Btw, if no one is working on the Quality pages, I could take over these, research the topic more and add something like the figure above to the corresponding page.

Sounds good.  If possible, I think the graphic can just be a Markdown list (with indentations, i.e. tree) or table.

(And the quality pages structure or section structure can also reflect it.)

Then it won't really need the graphic.

(Graphics are hard to edit, translate etc.)

Reply:
> 

Aaa, I see your point about pictures being hard to edit - I was also a bit worried about it. Therefore, I think the pictures definitely should not contain any fast-changing content (e.g., specific metrics in this case). But can they just reflect the categorisation? I.e., in the picture above remove the specific metrics but keep the structure - this is very likely to be static. To make the pictures editable, I could put the .pptx source along with the picture - in this case, anyone would be able to edit and make a new one. Not as easily as with plain markdown, but for the foundational stuff this might be ok. And each of the figures will still have an accompanying text - this can be machine-translated. What do you think?

Unfortunately, I don't know a way of making the structure above with just markdown lists - can one make several branches as on the figure? If one can, can you link me to an example of such a thing?

Reply:
I think one important item we should consider when thinking about images, besides its translation capabilities, is how they will fit on the website. Please note that the article content section has a somewhat fixed width, and anything that may exceed that width would require readers to left-right scroll. This would also affect phone navigability.

I agree with you that copying the image structure with markdown can be a bit complicated. I think that we may break it into different lists? Or perhaps we can style a table, although I'm uncertain if that is possible.

Reply:
Yes, we definitely need to think how it'll look on mobile devices!
Re width: can't we just set image width in percentage of the screen width rather than a fixed absolute width value? In html, this works well.

Reply:
Hi Lena!
Should we create a new issue regarding this discussion?

Reply:
Here it is: https://github.com/machinetranslate/machinetranslate.org/issues/231